tasks/system_probe.py

```diff
@@ -348,13 +348,19 @@ def clang_tidy(ctx, fix=False, fail_on_issue=False):
     network_bpf_dir = os.path.join(".", "pkg", "network", "ebpf")
     network_c_dir = os.path.join(network_bpf_dir, "c")
     network_files = list(base_files)
-    network_files.extend(glob.glob(network_c_dir + "/**/*.c"))
+    network_files.extend(glob.glob(network_c_dir + "/**/*[!http].c"))
     network_flags = list(build_flags)
     network_flags.append(f"-I{network_c_dir}")
     network_flags.append(f"-I{os.path.join(network_c_dir, 'prebuilt')}")
     network_flags.append(f"-I{os.path.join(network_c_dir, 'runtime')}")
     run_tidy(ctx, files=network_files, build_flags=network_flags, fix=fix, fail_on_issue=fail_on_issue)
 
+    http_files = [os.path.join(network_c_dir, 'prebuilt', 'http.c')]
```

Comment:
we shall run clang-tidy foreach of our prog {dns,conntrack,http,tracer} and with there respective {prebuilt,runtime} path (if exist)